### PR TITLE
Port project over to GPLv3

### PR DESCRIPTION
**Pull request by ShellyHerself**
Apply GPLv3 license notices to all files with code in them.

Btw, remember how I work with an editor that trims trailing whitespace? Yeah...

### CODE REVIEW

File: os_v4_hek/defs/weap.py [hunk 1]
```diff
@@ -1,4 +1,11 @@
-from ...os_v3_hek.defs.weap import *
```
Comment (MosesofEgypt) [inline os_v4_hek/defs/weap.py:1]: This import should not be removed.

**Review by MosesofEgypt — CHANGES_REQUESTED** [review]
os_v4_hek.defs.weap is missing an import

File: os_v4_hek/defs/weap.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 # See LICENSE for more information.
 #
 
+from .weap import *
```
Comment (MosesofEgypt) [inline os_v4_hek/defs/weap.py]: This is a recursive self import. It needs to be

```python
from ...os_v3_hek.defs.weap import *
```

**Review by MosesofEgypt — CHANGES_REQUESTED** [review]

File: os_hek/handler.py [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+# This file is part of Reclaimer.
+#
+# For authors and copyright check AUTHORS.TXT
+#
+# Reclaimer is free software under the GNU General Public License v3.0.
+# See LICENSE for more information.
+#
+
 from pathlib import Path
-import os
```
Comment (Mimickal) [inline os_hek/handler.py:2]: Is this an unused import?

File: shadowrun_prototype/handler.py [hunk 4]
```diff
@@ -1,5 +1,13 @@
...
+# This file is part of Reclaimer.
+#
+# For authors and copyright check AUTHORS.TXT
+#
+# Reclaimer is free software under the GNU General Public License v3.0.
+# See LICENSE for more information.
+#
+
 from pathlib import Path
-import os
```
Comment (Mimickal) [inline shadowrun_prototype/handler.py:2]: Unused import?

File: stubbs/handler.py [hunk 5]
```diff
@@ -1,5 +1,13 @@
...
+# This file is part of Reclaimer.
+#
+# For authors and copyright check AUTHORS.TXT
+#
+# Reclaimer is free software under the GNU General Public License v3.0.
+# See LICENSE for more information.
+#
+
 from pathlib import Path
-import os
```
Comment (Mimickal) [inline stubbs/handler.py:2]: Same thing?

File: os_hek/handler.py (hunk 3 above)
Comment (MosesofEgypt) [reply os_hek/handler.py:2]: yes

File: shadowrun_prototype/handler.py (hunk 4 above)
Comment (MosesofEgypt) [reply shadowrun_prototype/handler.py:2]: double yes

File: stubbs/handler.py (hunk 5 above)
Comment (MosesofEgypt) [reply stubbs/handler.py:2]: tr-tr-tr-triple yes

**Review by MosesofEgypt — APPROVED** [review]
Changes look good

**Pull request merged**